hazelcast/src/main/java/com/hazelcast/config/ConfigXmlGenerator.java

```diff
@@ -825,6 +827,29 @@ private void hotRestartXmlGenerator(StringBuilder xml, Config config) {
         xml.append("</hot-restart-persistence>");
     }
 
+    private void nativeMemoryXmlGenerator(StringBuilder xml, Config config) {
+        NativeMemoryConfig nativeMemoryConfig = config.getNativeMemoryConfig();
+        if (nativeMemoryConfig == null) {
+            xml.append("<native-memory enabled=\"false\" />");
+            return;
+        }
+        xml.append("<native-memory enabled=\"").append(nativeMemoryConfig.isEnabled())
+                .append("\"")
+                .append(" allocator-type=\"").append(nativeMemoryConfig.getAllocatorType())
+                .append("\"")
+                .append(">");
+        xml.append("<size")
+                .append(" unit=\"").append(nativeMemoryConfig.getSize().getUnit()).append("\"")
+                .append(" value=\"").append(nativeMemoryConfig.getSize().getValue()).append("\"")
+                .append("/>");
+
+        appendNode(xml, "min-block-size", nativeMemoryConfig.getMinBlockSize());
+        appendNode(xml, "page-size", nativeMemoryConfig.getPageSize());
+        appendNode(xml, "metadata-space-percentage", nativeMemoryConfig.getMetadataSpacePercentage());
```

Comment:
ok. so it won't use the notation with mantissa & exponent. 
I wasn't sure about it, thanks!